clang/lib/CIR/CodeGen/CIRGenBuilder.h

```diff
@@ -197,6 +199,10 @@ class CIRGenBuilderTy : public mlir::OpBuilder {
     return mlir::cir::PointerType::get(getContext(), ty);
   }
 
+  mlir::cir::PointerType getVoidPtrTy(unsigned AddrSpace = 0) {
+    return typeCache.VoidPtrTy;
```

Comment:
We should unreachable against the addrspace here.

Reply:
Done!